HTML/1. intro.md

```diff
@@ -0,0 +1,39 @@
+# Intro
+
+![HTML](https://img.17qq.com/images/mhsgsnffsky.jpeg)
+
+HTML is a markup language that is used to create webpages, so it's not like a programming language like Python
+___
+HTML is structured by tags and content which make up elements. There are 2 types of HTML Elements.
```

Comment:
They aren't different types of elements, they're all elements. It's just the syntax that is different.